src/components/note/note.style.ts

```diff
@@ -53,13 +53,11 @@ const StyledInlineControl = styled.div`
 const StyledTitleWrapper = styled.div`
   ${VARIANT_TYPES.map(
     (variant) => `
-      ${variant}{
-        font-weight: 700;
-        font-size: 16px;
-        line-height: 21px;
-        padding-bottom: 16px;
-      }
-    `,
+${variant}{
```

Comment:
comment: should set the styles to default to those of h3 now

If we are to still support this behaviour then this should be defaulted to the h3 styling (previously it was h2 styling). It might be worth speaking to the team to ensure that we want to keep this behaviour as it's not completely clear from the ticket.

We also need to ensure the DS is updated as this no longer matches what is supposed to be the source of truth. So, it could be worth speaking to @harpalsingh to make sure that gets updated if this is to be correct. I'd be reluctant to approve this until it does match the source of truth.

Reply:
In fact, upon reading the ticket again it seems it was mentioned that the override styling for the Node that is passed in should be completely removed.

`Should remove the Note specific H3 styling, if user passes text, render a h3 typography components, otherwise render the Node`

Could we please remove that styling block in that case and prompt @harpalsingh to update the Design System if Note should use H3 so Designers and Developers are both using the same styles?